stream/src/main/java/org/apache/kafka/stream/KStreamProcess.java

```diff
@@ -68,22 +66,18 @@
     private final ProcessorConfig config;
     private final Object lock = new Object();
     private final KStreamThread[] threads;
-    private final Set<String> topics;
-
 
-    public KafkaStreaming(KStreamTopology topology, StreamingConfig streamingConfig) {
 
-        if (streamingConfig.timestampExtractor() == null)
+    public KStreamProcess(Class<? extends PTopology> topologyClass, ProcessorProperties processorProperties) throws Exception {
```

Comment:
This class is actually not part of KStream, right? It is part of the processor package. Rather the connection between KStream and the processor layer is that KStream produces a topology...?